bibliopixel/project/project.py

```diff
@@ -44,10 +45,44 @@ def make_animation(layout, animation, run=None):
     return animation
 
 
-def _make_project(path=None, animation=None, run=None, **kwds):
+def make_remote(layout, remote):
+    OFF_ANIM = {
+        'typename': 'bibliopixel.animation.off.OffAnim'
+    }
+
+    animations = remote.get('animations', None)
```

Comment:
Always been that way!  The other Python linter (can't remember the name) complains about it endlessly in code...